src/mod_push_service_mongoosepush.erl

```diff
@@ -127,20 +128,22 @@ http_notification(Host, Method, URL, ReqHeaders, Payload) ->
                 StatusCode when StatusCode >= 200 andalso StatusCode < 300 ->
                     ok;
                 410 ->
-                    ?WARNING_MSG("issue=unable_to_submit_push_notification, https_status=410, reason=device_not_registered", []),
```

Comment:
I wonder where these discrepancies in using `http/https` came from, should we take care of it?